gamedata.cpp

```diff
@@ -56,6 +56,16 @@ ItemType id[] =
 	 -1, 0, 0,
 	 0, NULL, 0,
 	 "", { "", "", "", "" }, 0, 0},
+	{"human", "humans", "HUMN",
```

Comment:
You could just rename IT_MAN?

Reply:
I was trying to do this, had some issues with this before. Will try to make it this way, will update PR.